snuba/datasets/storages/errors_common.py

```diff
@@ -136,6 +136,7 @@
         project_column="project_id", replacer_state_name=ReplacerState.ERRORS,
     ),
     MappingColumnPromoter(mapping_specs={"tags": promoted_tag_columns}),
+    EventIdColumnProcessor(),
```

Comment:
I never noticed this, but I think this is in the wrong place same for transactions).  It feels this transformation should be a translator rule as the query produced by the translator does not have types that match the ones the logical query expects. Not blocking because transactions already work this way. Mind adding it to the backlog, please?